packages/hub/readme.md

```diff
@@ -39,13 +39,15 @@ $ yarn hub:watch (will rebuild app on file change)
 
 ### Interacting with the hub from a browser
```

Comment:
> a browser
Could be "from a `wallet`" to be precise

Reply:
It's a bit tricky, as from a user's perspective, the interaction is happening from rps. Also, rps is the module sending messages to the hub.  What about "Establishing a virtual channel between clients through the hub"?

Reply:
2255c2c49